flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobCacheSuccessTest.java

```diff
@@ -92,15 +100,15 @@ private void uploadFileGetTest(final Configuration config, boolean cacheWorksWit
 		BlobCache blobCache = null;
 		BlobStoreService blobStoreService = null;
 		try {
-			final Configuration cacheConfig;
-			if (cacheHasAccessToFs) {
-				cacheConfig = config;
-			} else {
-				// just in case parameters are still read from the server,
-				// create a separate configuration object for the cache
-				cacheConfig = new Configuration(config);
+			final Configuration cacheConfig = new Configuration(config);
+			cacheConfig.setString(BlobServerOptions.STORAGE_DIRECTORY,
+				temporaryFolder.newFolder().getAbsolutePath());
+			if (!cacheHasAccessToFs) {
+				// make sure the cache cannot access the HA store directly
+				cacheConfig.setString(BlobServerOptions.STORAGE_DIRECTORY,
+					temporaryFolder.newFolder().getAbsolutePath());
```

Comment:
seems that it wasn't used to its full extension - also the meaning of this variable was not what clear